wfl/Profiles/FoodOrderProfile.cs

```diff
@@ -60,6 +44,51 @@ protected override void Configure(EntityConfiguration<FoodOrder> configuration)
                             o => rc => rc.GetByIdAsync(o.RestaurantId)
                         )
                     )
+                )
+                .When(
```

Comment:
Keep in mind that Hypermedia is not security.  Even if something is not exposed via Hypermedia, it can still be called on the API.  We need to ensure that the controller method itself implements the security.

Along those lines, we should also think about how to consolidate the logic which verifies when you can take certain actions.  We wouldn't want the same logic being spread amongst too many locations.  Do you have any thoughts on how to handle this?